cli/flox/doc/flox-pull.md

```diff
@@ -7,53 +7,72 @@ header: "flox User Manuals"
 
 # NAME
 
-flox-pull -
+flox-pull - pull environment from FloxHub
 
 # SYNOPSIS
 
-flox [ `<general-options>` ] pull [ `<options>` ] [ \--force ] [ ( -m | \--main) ]
+```
+flox [ <general-options> ] pull
+     [-d=<path>]
+     [-a]
+     [-r=<owner/name> | <owner/name> | [-f]]
```

Comment:
```suggestion
     [-r=<owner>/<name> | <owner>/<name> | [-f]]
```

Reply:
i guess that's also as copied from the `help`
to align `help` with this we'd need to pass `argument("owner>/<name")`, not sure if that's the _right_ format though 🤷🏼‍♂️